nordicsemi/__main__.py

```diff
@@ -910,7 +910,7 @@ def update_progress(progress=0):
     if global_bar:
         global_bar.update(progress)
 
-@cli.group(short_help='Perform a Device Firmware Update over, BLE, Thread, or serial transport given a DFU package (zip file).')
+@cli.group(short_help='Perform a Device Firmware Update over, ANT, BLE, Thread, or serial transport given a DFU package (zip file).')
```

Comment:
```suggestion
@cli.group(short_help='Perform a Device Firmware Update over serial, BLE, Thread, Zigbee or ANT transport given a DFU package (zip file).')
```
Ref comment for README.